### PR TITLE
document available host preflights types

### PR DESCRIPTION
**Pull request by sgalsaleh**
Image 2022-01-13 at 2 05 01 PM

### CODE REVIEW

**Review by ajp-io — DISMISSED** [review]

File: src/markdown-pages/create-installer/host-preflights/block-devices.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+path: "/docs/create-installer/host-preflights/block-devices"
+date: "2022-01-13"
+weight: 29
+linktitle: "Block Devices"
+title: "Block Devices"
+---
+ 
+The block devices host preflight check can be used to detect and validate the block devices attached to the machine.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/block-devices.md:9]:
```suggestion
You can use the block devices host preflight check to detect and validate the block devices attached to the machine.
```

File: src/markdown-pages/create-installer/host-preflights/block-devices.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+### Parameters
+
+The `blockDevices` collector accepts the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties).
+
+## Block Devices Analyzer
+
+The `blockDevices` analyzer supports multiple parameters for filtering:
+
+`includeUnmountedPartitions`: Include unmounted partitions in the analysis. Disabled by default.<br/>
+`minimumAcceptableSize`: The minimum acceptable size to filter the available block devices by during analysis. Disabled by default.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/block-devices.md:24]:
```suggestion
`minimumAcceptableSize`: The minimum acceptable size to filter the available block devices during analysis. Disabled by default.
```

File: src/markdown-pages/create-installer/host-preflights/cpu.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+path: "/docs/create-installer/host-preflights/cpu"
+date: "2022-01-13"
+weight: 25
+linktitle: "CPU"
+title: "CPU"
+---
+ 
+The CPU host preflight check can be used to detect and validate the total number of CPU cores on the machine.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/cpu.md:9]:
```suggestion
The CPU host preflight check is used to detect and validate the total number of CPU cores on the machine.
```

File: src/markdown-pages/create-installer/host-preflights/disk-usage.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+path: "/docs/create-installer/host-preflights/disk-usage"
+date: "2022-01-13"
+weight: 28
+linktitle: "Disk Usage"
+title: "Disk Usage"
+---
+ 
+The disk usage host preflight check can be used to detect and validate the usage of a disk on which a particular directory lives.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/disk-usage.md:9]:
```suggestion
You can use the disk usage host preflight check to detect and validate the usage of a disk for particular directories.
```

File: src/markdown-pages/create-installer/host-preflights/host-os.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+---
+path: "/docs/create-installer/host-preflights/operating-system"
+date: "2022-01-13"
+weight: 30
+linktitle: "Operating System"
+title: "Operating System"
+---
+ 
+The operating system host preflight check can be used to detect and validate the name and version of the OS installed on the machine.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/host-os.md:9]:
```suggestion
You can use the operating system host preflight check to detect and validate the name and version of the OS installed on the machine.
```

File: src/markdown-pages/create-installer/host-preflights/host-services.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+path: "/docs/create-installer/host-preflights/host-services"
+date: "2022-01-13"
+weight: 31
+linktitle: "Host Services"
+title: "Host Services"
+---
+ 
+The host services host preflight check can be used to detect and validate the status of certain host system services.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/host-services.md:9]:
```suggestion
You can use the host services host preflight check to detect and validate the status of certain host system services.
```

File: src/markdown-pages/create-installer/host-preflights/http.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
+---
+path: "/docs/create-installer/host-preflights/http"
+date: "2022-01-13"
+weight: 32
+linktitle: "HTTP"
+title: "HTTP"
+---
+ 
+The HTTP host preflight check can be used to validate that the machine is able to connect to a certain HTTP address.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/http.md:9]:
```suggestion
The HTTP host preflight check is used to validate that the machine is able to connect to a certain HTTP address.
```

File: src/markdown-pages/create-installer/host-preflights/http.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+## HTTP Collector
+
+The `http` collector will collect information about the ability to connect to the specified HTTP address.
+
+### Parameters
+
+In addition to the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties), the `http` collector accepts the following parameters:
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/http.md:17]:
```suggestion
In addition to the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties), the `http` collector also accepts the following parameters:
```

File: src/markdown-pages/create-installer/host-preflights/memory.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+path: "/docs/create-installer/host-preflights/memory"
+date: "2022-01-13"
+weight: 26
+linktitle: "Memory"
+title: "Memory"
+---
+ 
+The memory host preflight check can be used to detect and validate the total amount of memory on the machine.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/memory.md:9]:
```suggestion
The memory host preflight check is used to detect and validate the total amount of memory on the machine.
```

File: src/markdown-pages/create-installer/host-preflights/system-packages.md [hunk 10]
```diff
@@ -0,0 +1,173 @@
+---
+path: "/docs/create-installer/host-preflights/system-packages"
+date: "2022-01-13"
+weight: 33
+linktitle: "System Packages"
+title: "System Packages"
+---
+
+The system packages host preflight check can be used to validate that certain packages are installed for a specific operating system.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/system-packages.md:9]:
```suggestion
The system packages host preflight check is used to validate that certain packages are installed for a specific operating system.
```

File: src/markdown-pages/create-installer/host-preflights/system-packages.md [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+
+## System Packages Collector
+
+The `systemPackages` collector will collect information about the specified host system packages depending on the operating system the packages are listed under.
+The collector will automatically detect the operating system ID/name and the version, and check if it has a match for that operating system ID/name (and optionally the version) in the spec. If so, it will collect information about the packages specified in the spec.
+
+This collector does not require sudo privileges.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/system-packages.md:16]:
```suggestion
Note: This collector does not require sudo privileges.
```

File: src/markdown-pages/create-installer/host-preflights/tcp-connect.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
+---
+path: "/docs/create-installer/host-preflights/tcp-connect"
+date: "2022-01-13"
+weight: 34
+linktitle: "TCP Connect"
+title: "TCP Connect"
+---
+ 
+The TCP connect host preflight check can be used to validate that the machine is able to connect to a certain TCP address.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/tcp-connect.md:9]:
```suggestion
The TCP connect host preflight check is used to validate that the machine is able to connect to a certain TCP address.
```

File: src/markdown-pages/create-installer/host-preflights/tcp-load-balancer.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
+---
+path: "/docs/create-installer/host-preflights/tcp-load-balancer"
+date: "2022-01-13"
+weight: 36
+linktitle: "TCP Load Balancer"
+title: "TCP Load Balancer"
+---
+ 
+The TCP load balancer host preflight check can be used to check and validate the connection to a specified TCP load balancer.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/tcp-load-balancer.md:9]:
```suggestion
The TCP load balancer host preflight check is used to check and validate the connection to a specified TCP load balancer.
```

File: src/markdown-pages/create-installer/host-preflights/tcp-port.md [hunk 14]
```diff
@@ -0,0 +1,72 @@
+---
+path: "/docs/create-installer/host-preflights/tcp-port-status"
+date: "2022-01-13"
+weight: 35
+linktitle: "TCP Port Status"
+title: "TCP Port Status"
+---
+ 
+The TCP port status host preflight check can be usee to detect the status of a TCP port to check whether it's open and available.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/tcp-port.md:9]:
```suggestion
The TCP port status host preflight check is used to detect the status of a TCP port to check whether it's open and available.
```

File: src/markdown-pages/create-installer/host-preflights/time.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
+---
+path: "/docs/create-installer/host-preflights/time"
+date: "2022-01-13"
+weight: 27
+linktitle: "Time"
+title: "Time"
+---
+ 
+The time host preflight check can be used to check the timezone of the system clock and whether the system clock is synchronized.
```
Comment (samantha-replicated) [inline src/markdown-pages/create-installer/host-preflights/time.md:9]:
```suggestion
The time host preflight check is used to check the timezone of the system clock and whether the system clock is synchronized.
```

**Review by samantha-replicated — APPROVED** [review]
Committed a few minor edits. Otherwise, LGTM! Approving.

**Pull request merged**